pkg/server/api/status.go

```diff
@@ -16,10 +16,10 @@ func CreateStatus(code codes.Code, format string, a ...interface{}) *types.Statu
 	}
 }
 
-// StatusFromError creates a proto Status from given error, if no error OK status is returned
+// StatusFromError creates a proto Status from given error
 func StatusFromError(err error) *types.Status {
 	if err == nil {
-		return &types.Status{Code: int32(codes.OK)}
+		return nil
```

Comment:
Why change this?

Reply:
the idea is to remove that as part of refactor on new approach, I can keep it if you prefer